src/api/libraryApi.ts

```diff
@@ -1,19 +1,24 @@
 import { homeUrl, librariesBaseUrl } from "../constants/ApiConstants";
 import Library from "../interfaces/Library";
+import LibraryPreview from "../interfaces/LibraryPreview";
 
-// TODO: will we eventually want this to be get all libraries for a single user? Will have to update backend endpoints
-export function getAllLibraries(): Promise<Library[]> {
-    return fetch(homeUrl + librariesBaseUrl, {
+export function getAllLibrariesWithGames(): Promise<Library[]> {
+    return fetch(homeUrl + librariesBaseUrl + "withGames/", {
         method: "GET",
     })
-    .then((response) => response.json());
+    .then((response) => {
+        console.log(response);
```

Comment:
We should figure out how we want to do logging in the future. We could get a logger library if we want, not sure if Kibana is something we can get for free or cheap but maybe down the line? (i also vaguely hate kibana so not opposed to not using it lol)

Reply:
Definitely on board with discussing this! Created a ticket to track it https://gaming-backlog.atlassian.net/browse/GB-61

Reply:
Oops also realizing I did not mean to leave that log in there! Will remove it in my next PR